django/src/ft_auth/views.py

```diff
@@ -68,6 +70,18 @@ def password_update(request: HttpRequest):
 	logger.info(f"Updated password of {request.user.username}.")
 	return HttpResponse(status=200)
 
+def upload_avatar(request):
+ 
+	if request.FILES.get('avatar'):
```

Comment:
Need to protect file size here too

Reply:
adressed in : [d42776c](https://github.com/TheSecretOrganization/ft_transcendence/pull/247/commits/d42776cdca442763caa7de3c21eacf49a904bcf6)